website/blog/2023-12-29-AgentDescriptions/index.mdx

```diff
@@ -0,0 +1,139 @@
+---
+title: "All About Agent Descriptions"
+authors:
+  - afourney
+tags: [AutoGen]
+---
+
+
+## TLDR
+
+AutoGen 0.2.2 introduces a [description](https://microsoft.github.io/autogen/docs/reference/agentchat/conversable_agent#__init__) field to ConversableAgent (and all subclasses), and changes GroupChat so that it uses agent `description`s rather than `system_prompt`s when choosing which agents should speak next.

```

Comment:
system_prompt -> system_message
to match the code

Reply:
Thanks. Fixed here, and throughout.